src/components/Select/MultiSelect.scss

```diff
@@ -0,0 +1,66 @@
+@import '~om';
+
+.MultiSelect {
+  $self: &;
+
+  min-height: 2.5625rem;
+  position: relative;
+  display: flex;
+  width: 100%;
+
+  &__search {
+    @include input-styles();
+    display: flex;
+
+    &--error {
+      border: 1px solid color(code-red);
+    }
+  }
+
+  &__selected-items {
+    display: flex;
+    gap: 0.3rem;
+    max-width: 80%;
+    overflow: scroll;
+  }
+
+  &__selected-item {
+    display: flex;
+    align-items: center;
+    border: 1px solid black;
+    border-radius: 0.6rem;
```

Comment:
Can this also use a `space` value? Maybe `space(2)` or `space(3)`?

Reply:
I've seen this in a couple of places.  Maybe there should be a "radius" function in om, that covers the cases that we've been using for border-radius values.  It feels weird to use `space` for this, since it's not actually the space between elements.  The values can map to valid spacing distances, but we wouldn't have to think about it.